projects/tinkerbell/hegel/docker/linux/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install update upgrade -y shadow-utils ca-certificates
```

Comment:
I can show you a better way to handle this.  This can be just off the minimal-base.